src/mongocrypt-key-broker.c

```diff
@@ -1087,11 +1156,15 @@ _mongocrypt_key_broker_add_test_key (_mongocrypt_key_broker_t *kb,
    key_returned_t *key_returned;
    _mongocrypt_key_doc_t *key_doc;
 
-   BSON_ASSERT (kb);
+   BSON_ASSERT_PARAM (kb);
+   BSON_ASSERT_PARAM (key_id);
+
    key_doc = _mongocrypt_key_new ();
+   BSON_ASSERT (key_doc);
```

Comment:
```suggestion
```
Redundant assertion: `_mongocrypt_key_new ()` always returns a not-null pointer.